### ISSUE TITLE
Missing plugin: unable to count total tracks when per_disc_numbering is on

### ISSUE THREAD

**Issue by kooimens** [sampsyo/beets#1309]
I'm using the missing plugin to get information about the total amount of missing tracks per album (config option, count = yes). However, the command `beet missing` then returns every album that has more than one disc. The missing track total equals the amount of all tracks that are not on disc one.

**Comment by djl** [comment]
:+1:

This has been annoying me, too.

**Comment by djl** [comment]
Hmm. While digging deeper into this I may have accidentally fixed the issue on my end.

It seems that if you have `per_disc_numbering` set to `False` when you imported an album with multiple discs, then `item.tracktotal` for all discs will be set to the track count for the first disc. If I set `per_disc_numbering` before re-importing an album the track totals were fixed.

I say I "may" have fixed the issue because I'm 99% sure I've always had `per_disc_numbering` set :confused:
